revisions

fivetran-catfritz opened the pull request:
**Are you a current Fivetran customer?** 
<!--- Please tell us your name, title and company -->

**What change(s) does this PR introduce?** 
<!--- Describe what changes your PR introduces to the package and how to leverage this new feature. -->

**Did you update the CHANGELOG?** 
<!--- Please update the new package version’s CHANGELOG entry detailing the changes included in this PR. -->
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [ ] Yes

**Does this PR introduce a breaking change?**
<!--- Does this PR introduce changes that will cause current package users' jobs to fail or require a `--full-refresh`? -->
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [ ] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 
<!--- The dbt_project.yml and the integration_tests/dbt_project.yml files contain the version number. Be sure to upgrade it accordingly -->
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [ ] Yes

**Is this PR in response to a previously created Bug or Feature Request**
<!--- If an Issue was created it is helpful to track the progress by linking it in the PR. -->
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [ ] Yes, Issue/Feature [link bug/feature number here]
- [ ] No 

**How did you test the PR changes?** 
<!--- Proof of testing is required in order for the PR to be approved. -->
<!--- To check a box, remove the space and insert an x in the box (eg. [x] Buildkite). --> 
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [ ] Buildkite <!--- Buildkite testing is only applicable to Fivetran employees. --> 
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
<!--- To check a warehouse remove the space and insert an x in the box (eg. [x] Bigquery). --> 
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [ ] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
<!--- For a complete list of markdown compatible emojis check our this git repo (https://gist.github.com/rxaviers/7360908)  --> 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

fivetran-joemarkiewicz commented on models/recharge__charge_line_item_history.sql:
```diff
@@ -93,10 +93,10 @@ with charges as (
         row_number() over(partition by unioned.charge_id 
             order by unioned.line_item_type, unioned.index) 
-            as charge_row_num, --thinking of this to replace index.
+            as charge_row_num, -- CLARIFY IN DOCS
```
Reminder to remove after updating the docs.

fivetran-catfritz replied:
done

fivetran-joemarkiewicz reviewed (approved):
@fivetran-catfritz great job reworking the MRR report to take into consideration the subscriptions prepaid and rendered over time! I have just two very minor comments about documenting a field, but everything else looks great and validated on my end!

fivetran-catfritz commented on models/recharge.yml:
```diff
@@ -75,19 +75,26 @@ models:
-      - name: index
-        description: A unique numeric row produced for every concurrent charge_id.
+      - name: charge_row_num
+        description: EXPLAIN
```
Updated.

fivetran-catfritz commented:
@fivetran-joemarkiewicz I have updated the docs (the "Files changed" suppresses recharge.yml) and renamed some columns for clarity. It is ready for re-review!

fivetran-catfritz commented on dbt_project.yml:
```diff
@@ -6,7 +6,6 @@ require-dbt-version: [">=1.3.0", "<2.0.0"]
 
 vars:
   recharge:
-    dbt_date:time_zone: "UTC"
```
Note--this was leftover from some prior experiments but unnecessary now.

fivetran-catfritz commented on models/intermediate/int_recharge__customer_details.sql:
```diff
@@ -11,15 +11,12 @@ with customers as (
+        round(cast(sum(order_item_quantity) as {{ dbt.type_numeric() }}), 2) as total_quantity_ordered,
         round(cast(avg(order_item_quantity) as {{ dbt.type_numeric() }}), 2) as avg_item_quantity_per_order,
         round(cast(sum(order_line_item_total) as {{ dbt.type_numeric() }}), 2) as total_order_line_item_total,
-        round(cast(avg(order_line_item_total) as {{ dbt.type_numeric() }}), 2) as avg_order_line_item_total,
-        round(cast(sum(total_tax) as {{ dbt.type_numeric() }}), 2) as total_amount_taxed,
-        round(cast(sum(total_discounts) as {{ dbt.type_numeric() }}), 2) as total_amount_discounted,
-        round(cast(sum(total_refunds) as {{ dbt.type_numeric() }}), 2) as total_refunds,
-        count(case when lower(billing.order_type) = 'checkout' then 1 else null end) as total_one_time_purchases
```
These were shifted to the charges cte to make clear the differentiation between an order or charge source.

fivetran-joemarkiewicz commented:
@fivetran-catfritz Looks good to merge!
